themes/bootstrap3/templates/combined/results.phtml

```diff
@@ -17,7 +17,7 @@
       'searchId' => $this->results->getSearchId(),
       'searchClassId' => $this->params->getSearchClassId(),
       'checkboxFilters' => $this->params->getCheckboxFacets(),
-      'filterList' => $this->params->getFilters(),
+      'filterList' => $this->params->getFilterList(),
```

Comment:
How much time did you spend investigating why `getFilters` is used in some places and `getFilterList` in others? This inconsistency seems like it may indicate some pre-existing bugs. I haven't taken time to do a deep dive yet, but I'm happy to spend time on it next week if you haven't already. I also wonder if it might be wise to deprecate `getFilters` and rename it `getRawFilters` to better differentiate between these methods.

Reply:
Practically no time at all. I'll take another look.

Reply:
Looks like other places where getFilters is used make sense. I posted #1489 to add getRawFilters and deprecate getFilters.

Reply:
Thanks!